cloudvolume/datasource/graphene/__init__.py

```diff
@@ -49,7 +50,10 @@ def create_graphene(
     )
 
     mesh = GrapheneMeshSource(meta, cache, config)
-    # skeleton = PrecomputedSkeletonSource(meta, cache, config)
+    try:
```

Comment:
This a little over-defensive imo. If Skeletons are crashing because of the dang info file, 1) I want to know about it 2) You're probably going to see issues elsewhere 3) This only covers 1 of 5 potential protocols.

Reply:
I take this back slightly, I still feel this is overly defensive because errors should be handled by the skeleton source. However, graphene really only uses the https protocol.